### PR TITLE
Addresses issue #54

### PR DESCRIPTION
**Pull request by kelseycribari**

### CODE REVIEW

File: src/listener/server-hardware.js [hunk 1]
```diff
@@ -23,9 +23,13 @@ THE SOFTWARE.
 import Listener from './base-listener';
 import { buildD3Chart } from '../charting/chart';
 import { getLogicalInterconnectsMap } from '../ov-brain';
+//import the getDevicename from the lexer class to get the name of the blades
```
Comment (danielreed) [inline src/listener/server-hardware.js]: You probably don't want this here.

File: src/listener/server-profiles.js [hunk 2]
```diff
@@ -21,6 +21,8 @@ THE SOFTWARE.
 */
 
 import Listener from "./base-listener";
+import { getDeviceName } from '../middleware/utils/lexer';
+//import { transformHyperlinks } from '../oneview-sdk/utils/enhance';
```
Comment (danielreed) [inline src/listener/server-profiles.js]: Probably remove this too.

File: src/ov-brain.js [hunk 3]
```diff
@@ -32,9 +32,9 @@ export default class OneViewBrain {
       if (res && res.members) {
         for (var i = 0; i < res.members.length; i++) {
           const sh = res.members[i];
-          Lexer.addNamedDevice(sh.name, sh.uri, 'name');
-          Lexer.addNamedDevice(sh.serialNumber, sh.uri, 'serialNumber');
-          robot.brain.set("__hpe__" + sh.uri, {uri:sh.uri, keys:{name:sh.name, serialNumber: sh.serialNumber}});
+          Lexer.addNamedDevice(sh.name, sh.uri, 'name', sh.hyperlink, robot);
```
Comment (danielreed) [inline src/ov-brain.js]: Why are you passing the robot here, is it needed by the addNamedDevice function?  Same question below.

File: src/listener/server-hardware.js [hunk 4]
```diff
@@ -23,9 +23,12 @@ THE SOFTWARE.
 import Listener from './base-listener';
 import { buildD3Chart } from '../charting/chart';
 import { getLogicalInterconnectsMap } from '../ov-brain';
+import { getDeviceName } from '../middleware/utils/lexer';
```
Comment (danielreed) [inline src/listener/server-hardware.js]: Is the function called getDeviceNameAndHyperLink now?  This import will have to be changed to match.

File: src/ov-brain.js (hunk 3 above)
Comment (kelseycribari) [inline src/ov-brain.js]: I was just passing it for debugging so I could print to the console. I just forgot to take that one out. I'll fix it.

**Comment by jonmccormick** [comment]
Good

**Comment by jonmccormick** [comment]
Good

**Pull request merged**